gatsby-config.js

```diff
@@ -2,120 +2,132 @@ const urljoin = require('url-join');
+    siteMetadata: {
+        siteUrl: urljoin(config.siteUrl, config.pathPrefix),
+        rssMetadata: {
+            site_url: urljoin(config.siteUrl, config.pathPrefix),
+            feed_url: urljoin(config.siteUrl, config.pathPrefix, config.siteRss),
+            title: config.siteTitle,
+            description: config.siteDescription,
+            image_url: `${urljoin(
+                config.siteUrl,
+                config.pathPrefix,
+                '/images/cover.png'
+            )}`,
+            copyright: config.copyright,
+        },
     },
-    {
-      resolve: "gatsby-source-filesystem",
-      options: {
-        name: "posts",
-        path: `${__dirname}/content/posts/`
-      }
+    mapping: {
+        'MarkdownRemark.frontmatter.author': `authors`,
     },
-    {
-      resolve: 'gatsby-transformer-remark',
-      options: {
-        plugins: [
-          {
-            resolve: 'gatsby-remark-images',
+    plugins: [{
+            resolve: `gatsby-transformer-yaml`,
             options: {
-              maxWidth: 690,
-              linkImagesToOriginal: false,
+                typeName: ({ node, object, isArray }) => object.type,
             },
-          },
-
-          {
-            resolve: 'gatsby-remark-responsive-iframe',
-          },
-          'gatsby-remark-prismjs',
-          {
-            resolve: 'gatsby-remark-copy-linked-files',
-            options: {},
-          },
-        ],
-      },
-    },
-    {
-      resolve: "gatsby-plugin-google-tagmanager",
-      options: {
-        id: config.googleTagManagerId,
-      },
-    },
-    {
-      resolve: 'gatsby-plugin-nprogress',
-    },
+        },
+        'gatsby-plugin-react-helmet',
+        'gatsby-plugin-lodash',
+        `gatsby-plugin-styled-components`,
+        `gatsby-transformer-sharp`,
+        'gatsby-plugin-sharp',
+        {
+            resolve: 'gatsby-source-filesystem',
+            options: {
+                name: 'assets',
+                path: `${__dirname}/static/`,
+            },
+        },
+        {
+            resolve: `gatsby-source-filesystem`,
+            options: {
+                name: 'news',
+                path: `${__dirname}/content/news.yaml`,
+            },
+        },
+        {
+            resolve: 'gatsby-source-filesystem',
+            options: {
+                name: "posts",
+                path: `${__dirname}/content/authors.yaml`
+            }
+        },
+        {
+            resolve: "gatsby-source-filesystem",
+            options: {
+                name: "posts",
+                path: `${__dirname}/content/posts/`
+            }
+        },
+        {
+            resolve: 'gatsby-source-filesystem',
+            options: {
+                name: "press",
+                path: `${__dirname}/content/authors.yaml`
+            }
+        },
+        {
+            resolve: "gatsby-source-filesystem",
+            options: {
+                name: "press",
```

Comment:
Jména u `gatsby-source-filesystem` u všech pluginů nijak nevyužíváme. Vím, že je to pozůstatek, ale přidání tiskovek mne dovedlo k tomu ověřit, zda to jméno má nějaké využití a nemá. Mohu tě v rámci ostatních úprav poprosit o smazání?